ruby_version_file_parser_test.go

```diff
@@ -0,0 +1,99 @@
+package bundleinstall_test
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"testing"
+
+	bundleinstall "github.com/paketo-buildpacks/bundle-install"
+	"github.com/sclevine/spec"
+
+	. "github.com/onsi/gomega"
+)
+
+const RUBY_VERSION_FILE_TEMPLATE = "%s\n"
+
+func testRubyVersionFileParser(t *testing.T, context spec.G, it spec.S) {
+	var (
+		Expect = NewWithT(t).Expect
+
+		path   string
+		parser bundleinstall.RubyVersionFileParser
+	)
+
+	it.Before(func() {
+		file, err := os.CreateTemp("", ".ruby-version")
+		Expect(err).NotTo(HaveOccurred())
+		defer file.Close()
+
+		path = file.Name()
+
+		parser = bundleinstall.NewRubyVersionFileParser()
+	})
+
+	it.After(func() {
+		Expect(os.RemoveAll(path)).To(Succeed())
+	})
+
+	context("ParseVersion", func() {
+		context("when given different types of versions", func() {
+			it("parses the versions correctly", func() {
+				versions := []string{
+					"2",
+					"2.6",
```

Comment:
I don't know if these should be valid in a `.ruby-version` but there's not really a spec for it out there so I just went with the same validation regex that is already used in the `Gemfile`.

Reply:
I'm fine with that